systers_portal/templates/meetup/base.html

```diff
@@ -1,14 +1,14 @@
 {% extends "base.html" %}
 
 {% block title %}
-  {{ object.name }}
+  {{ meetup_location.name|capfirst }}
```

Comment:
We don't need the capfirst. We expect that the admins who entered the info, know what the name of the meetup location should look like. Same on line 11.